src/store/record.js

```diff
@@ -32,8 +33,9 @@ let recordStore = {
         },
         sections: {
             generalInformation: initEditorSections(false, ["generalInformation"]).generalInformation,
+            organisations: {},
+            additionalInformation: initEditorSections(false, ["additionalInformation"]).additionalInformation,
             publications: initEditorSections(false, ["publications"]).publications,
```

Comment:
You shouldn't need to do this. I was doing it for the generalInformation tab because it's also re-used by the "create new record" page and needed to enforce 2 way dsata binding by pre-declaring the fields.
Look how both data_access and organisations are empty on dev.